.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
-node_js: 10
+node_js: 14
```

Comment:
Bumping node version fixes `FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory`